### ISSUE TITLE
Does mediapipe not support mobile browsers?

### ISSUE THREAD

**Issue by xielong** [iamhimanshu0/face-filter#2]
I saw this comment in the code: "Please use a Desktop/Laptop in order to run the filter". 
Does this mean that mediapipe does not support mobile browsers?

**Comment by iamhimanshu0** [comment]
yes it's support both mobile browser and Desktop browser, i didn't made my UI responsive that's why,
